ZenPacks/zenoss/LinuxMonitor/modeler/plugins/zenoss/cmd/linux/lvm.py

```diff
@@ -137,10 +134,13 @@ def process(self, device, results, log):
             elif section == 'NAME':
                 # device block can be 'vg_name-lv_name' or 'vg_name-lv_name (DM-X)' format
                 # depending on linux flavor
+                columns = {col[0]: col[1].strip() for col in columns.items()}
                 device_block = columns['device_block'].split()[0]
                 lsblk_dict[device_block] = {}
                 lsblk_dict[device_block]['mount'] = columns['mount']
                 lsblk_dict[device_block]['major_minor'] = columns['major_minor']
+                if columns['type'] in ('disk', 'lvm'):
```

Comment:
It looks like we should include partitions (part) here too. I'm not getting a mapping from my /dev/sda1 partition to its /boot filesystem.